packages/compass-e2e-tests/tests/atlas-cloud/global-writes.test.ts

```diff
@@ -0,0 +1,143 @@
+import { expect } from 'chai';
+import type { Compass } from '../../helpers/compass';
+import { cleanup, init, Selectors } from '../../helpers/compass';
+import type { CompassBrowser } from '../../helpers/compass-browser';
+import { createGeospatialCollection } from '../../helpers/insert-data';
+import {
+  DEFAULT_CONNECTION_NAMES,
+  isTestingAtlasCloudSandbox,
+} from '../../helpers/test-runner-context';
+
+type GeoShardingFormData = {
+  secondShardKey: string;
+  keyType?: 'UNIQUE' | 'HASHED';
+};
+
+type GeoShardingState = 'UNSHARDED' | 'SHARDING' | 'SHARD_KEY_CORRECT';
+
+async function createGeoShardKey(
+  browser: CompassBrowser,
+  formData: GeoShardingFormData
+) {
+  // shard-collection-form
+  await browser.setComboBoxValue(
+    Selectors.GlobalWrites.ShardKeyFormSecondKeyInputCombobox,
+    formData.secondShardKey
+  );
+
+  if (formData.keyType) {
+    await browser.clickVisible(
+      Selectors.GlobalWrites.ShardKeyFormAdvancedOptionsToggle
+    );
+    await browser.clickParent(
+      Selectors.GlobalWrites.shardKeyFormIndexType(formData.keyType)
+    );
+  }
+  await browser.clickVisible(Selectors.GlobalWrites.ShardKeyFormSubmitButton);
+}
+
+async function waitForGlobalWritesStatus(
+  browser: CompassBrowser,
+  nextState: GeoShardingState
+) {
+  await browser.waitUntil(async () => {
+    const content = await browser.$(
+      Selectors.GlobalWrites.tabStatus(nextState)
+    );
+    return await content.isDisplayed();
+  });
+}
+
+describe('Global writes', function () {
+  let compass: Compass;
+  let browser: CompassBrowser;
+
+  before(async function () {
+    compass = await init(this.test?.fullTitle());
+    browser = compass.browser;
+    await browser.setupDefaultConnections();
+  });
+
+  before(function () {
+    if (!isTestingAtlasCloudSandbox()) {
+      this.skip();
+    }
+  });
+
+  after(async function () {
+    await cleanup(compass);
+  });
+
+  it('should be able to shard an unsharded namespace and also unmanage it', async function () {
+    // Sharding a collection takes a bit longer
+    this.timeout(60_000);
```

Comment:
This is smaller than our default timeout in CI though

Reply:
🤭 
will remove it

Reply:
actually bumped it for whole suite